modules/drivers/athena/src/metabase/driver/athena/hive_parser.clj

```diff
@@ -0,0 +1,62 @@
+(ns metabase.driver.athena.hive-parser
+  (:require
+   [cheshire.core :as json]
+   [clojure.string :as str]
+   [clojure.walk :as walk]))
+
+(set! *warn-on-reflection* true)
+
+(defn- parse-to-json-string [s]
+  (loop  [schema s
+          closes []
+          result ""]
+    (cond
+      (not (nil? (re-find #"^array<[a-z0-9]*>" schema)))
+      (recur (str/replace-first schema #"^array<[a-z0-9]*>" "")
+             closes
+             (str result "[]"))
+
+      (str/starts-with? schema "array<")
+      (recur (str/replace-first schema #"array<" "")
+             (conj closes "]")
+             (str result "["))
+
+      (str/starts-with? schema "map<string,string>")
```

Comment:
I noticed that non string->string maps for example `map<int,int>` won't parse properly. so that may be a problem. I tracked down some info on this schema format here: https://gist.github.com/kzhangkzhang/258d18858889fa97194011a249b74c43#complexcollection-types and it looks like maps can even have complex values as their keys.

I can look into fixing this piece.